### PR TITLE
Polish translation for .ts files to reach 100%

### PR DESCRIPTION
**Pull request by MichalZr6**

### CODE REVIEW

File: mapeditor/translation/polish.ts [hunk 1]
```diff
@@ -1096,10 +1096,10 @@
...
         <location filename="../inspector/inspector.cpp" line="360"/>
         <source>%n tile(s)</source>
-        <translation type="unfinished">
-            <numerusform></numerusform>
-            <numerusform></numerusform>
-            <numerusform></numerusform>
+        <translation>
+            <numerusform>%n heks</numerusform>
+            <numerusform>%n heksy</numerusform>
+            <numerusform>%n heksów</numerusform>
```
Comment (IvanSavenko) [inline mapeditor/translation/polish.ts:1102]: Is this correct for Polish when referring to adventure map tiles? They are not hexagons, but squares / tiles.

Comment (MichalZr6) [reply mapeditor/translation/polish.ts:1102]: Heks might relate to a single, occupiable field in a wider context, however - as for adventure's map tiles - maybe "pole" is better. Does anyone have additional thoughts on that ?

Comment (dydzio0614) [reply mapeditor/translation/polish.ts:1102]: use "pole"

Comment (MichalZr6) [reply mapeditor/translation/polish.ts]: done

**Pull request merged**